index.html

```diff
@@ -188,17 +188,18 @@ <h3 class="customHeaderStyle">Sugarizer</h3>
         <div class="row">
             <div class="col-md-4">
                 <div class="border-box boxDownload"  >
-                    <h3 class="customHeaderStyle">OLPC XO</h3>
+                    <h3 class="customHeaderStyle">Trisquel</h3>
                     <figure>
-                    <img src="{{ site.baseurl }}/assets/computer.svg" class="span1 customLogoImageStyle" alt="computer svg">
+                    <img src="{{ site.baseurl }}/assets/Trisquel-logo.png" class="span1 customLogoImageStyle" alt="Flatpak svg">
                     </figure>
-                    <p>The full Sugar environment on
-                    an OLPC XO laptop.</p>
-                    <ul >
-                        <li class="listFont">
-                        <a href="http://wiki.laptop.org/go/Release_notes/13.2.9" class="hrefCustomColor">
-                            How to get Sugar on an OLPC XO Laptop</a></li>
-                    </ul>
+                    <p>The full Sugar environment
+                        installed on Trisquel GNU/Linux computer.</p>
+                        <ul >
+                            <li class="listFont"> Trisquel on a Sugar Toast a live/installable iso image that can be run from 
```

Comment:
Should read _Trisquel on a Sugar Toast is a live/installable_.

Reply:
@chimosky Could you please confirm? Do I need to remove lines 195-198, and there is only one line `<p>Trisquel on a Sugar Toast is a live/installable.</p>`?

Reply:
I'm saying just that part of the text needs to change, every other part is okay.